src/sass/components/_tour.scss

```diff
@@ -29,3 +29,37 @@
   }
 }
 /* stylelint-enable declaration-no-important */
+
+// Override background image
+.slds-popover_walkthrough .slds-popover__header {
+  background-image: url('~img/popover-header.png');
+}
+
+.popover-wrapper {
+  position: absolute;
+  top: -10px;
+
+  .slds-page-header__title & {
+    top: unset;
```

Comment:
@dvdherron I wonder if we can use a more specific or targeted class since this one applies to both the logo area and the title below it, but then the next class we see overrides this top value as well.

Reply:
changed this to 

`.global-header & {
  /* . . .*/
}`

and
 `
.page-header & {
/* .  . .*/
}
`
I think that's more clear regarding which parent elements we're targeting for the offset @stacyk ?

Reply:
@dvdherron I think this is now too general, and a slight regression.

1. We already have a `.page-header` class for the entire header, so now there are duplicate classes there. Are we sure we want this style to apply to any popover in the entire page header? I thought it was specific to the title.
2. Now the `.global-header` style is applying to the login/logout popover as well, but I don't think we actually want it there.

![Screen Shot 2021-04-23 at 10 46 47 AM](https://user-images.githubusercontent.com/552316/115888725-3ed5b100-a421-11eb-8df8-22041dc4e1ee.png)

![Screen Shot 2021-04-23 at 10 46 23 AM](https://user-images.githubusercontent.com/552316/115888735-4301ce80-a421-11eb-80f4-f2016b293af7.png)